tools/README.md

```diff
@@ -0,0 +1,31 @@
+## Tools
+
+## `generateImageAssets`
+
+Automatically updates your project dependencies to use a consistent [`react-native-bootsplash`](https://github.com/zoontek/react-native-bootsplash) compatible icon.
+
+** Be sure to commit your work before executing! **
+
+### Usage
+
+```bash
+node ./generateImageAssets.js --projectDir <path/to/project> --icon <path/to/icon>
+```
+
+This will update the icons of the React Native project defined at `path-to-project-directory` to the icon specified by `<path-to-icon>`. This tool currently relies on the naming conventions that are used in the [Example Project](https://github.com/zoontek/react-native-bootsplash/tree/master/example); and will therefore create the following files:
+
+```bash
+<path/to/project>/assets/react_logo.png (100x100)
```

Comment:
`bootsplash_logo` seems a better, more generic name (it was named `react_logo` because well…it's the react logo 😅). I would update the example file name accordingly 🙂

Reply:
Haha, _totally_ agreed. My heart went out to the projects which depended on the React logo!

So you're okay with enforcing a fixed name for the image asset? I think this would be great because it would allow us to develop lots of off-the-shelf transition styles.

Reply:
Completely OK to enforce `bootsplash_logo.png`!